config/bss-milestone-1-firebase-adminsdk-t5r8j-a10d0f2502.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
👋 Looks like you accidentally committed this file. This file is sensitive as it gives admin permissions to your Firebase account. Follow these 4 steps to revoke permissions for these credentials. You may also receive an email from Github, Google, or Firebase letting you know that these credentials are compromised.

<img width="990" alt="revoke-service-account" src="https://user-images.githubusercontent.com/53164431/158276294-6164651e-70bf-4e8f-98e7-dbb8cad22037.png">

<img width="1121" alt="gcp-iam-revoke" src="https://user-images.githubusercontent.com/53164431/158276292-474f59ff-5a4a-4ed7-be8e-4162557a0720.png">